eclair-core/src/main/resources/reference.conf

```diff
@@ -20,7 +20,7 @@ eclair {
   // override this with a script/exe that will be called everytime a new database backup has been created
   # backup-notify-script = "/absolute/path/to/script.sh"
 
-  watcher-type = "bitcoind" // other *experimental* values include "electrum"
+  watcher-type = "bitcoind"
```

Comment:
We could remove this. Will we add a new watcher type in a near future?

Reply:
Agreed, we can remove it. I don't think we'll support a new watcher type in the near future.

Reply:
Done in https://github.com/ACINQ/eclair/pull/1750/commits/9771ce6d4868ac7955f6d338dc9f70779d523f88